content/posts/0021-java-buildpacks-enhancements.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Java Buildpacks enhancements
+date: "2023-06-16"
+slug: java-buildpacks-enhancements
+author: anthonydahanne
+---
+
+During the last few weeks, some enhancements were brought to buildpacks related to the Java ecosystem.
+
+Let's discover what's new!
+
+### TL;DR
+
+Checkout the [samples repository for Java examples](https://github.com/paketo-buildpacks/samples/tree/main/java) and the [Paketo Buildpacks official documentation](https://paketo.io/docs/howto/java/) - those provide way more details about Paketo uses cases (native, certificates, network-less, etc.) than the different build plugins (Maven and Gradle) documentation.
+
+And they're always up-to-date! If the rare event they're not, create an issue or Pull Request, and we will fix that!
+
+## No more Gradle welcome message
+This enhancement was pretty trivial to add, but we hope you will like it: we have disabled Gradle build default welcome message.
+
+It's interesting to learn about Gradle new features in the welcome message, but when you re-build several times with the same gradle version, that does not remember you used it more than once already (because it's run from fresh containers), it can be get a bit repetitive.
+
+More details in the [PR](https://github.com/paketo-buildpacks/gradle/issues/244).
+
+## Additional build arguments for Gradle and Maven
+
+While before you needed to always provide the full arguments lists of arguments even if you just wanted to activate 1 parameter, using 
+* `BP_MAVEN_BUILD_ARGUMENTS` for Maven builds or
+* `BP_GRADLE_BUILD_ARGUMENTS` for Gradle builds
+
+Now you can just append additional arguments to the default BP_{MAVEN|GRADLE}_BUILD_ARGUMENTS, using:
+* `BP_MAVEN_ADDITIONAL_BUILD_ARGUMENTS` for Maven builds or
+* `BP_GRADLE_ADDITIONAL_BUILD_ARGUMENTS` for Gradle builds
+
+More details in the [Maven PR](https://github.com/paketo-buildpacks/maven/pull/265) and in the [Gradle PR](https://github.com/paketo-buildpacks/gradle/pull/238).
+
+## Specify Maven active profiles
+Very similar to the previous enhancement, but just for Maven this time, you can activate a profile without specifying the full list of Maven build arguments now, using:
+
+* `BP_MAVEN_ACTIVE_PROFILES`
+
+This enhancement ([PR](https://github.com/paketo-buildpacks/maven/pull/265)) was actually key for the next item in our list, Spring Boot native detection.
+
+
+## Better Spring Boot Native detection
+
+Since Spring Boot 3.0, Paketo buildpacks support Spring Boot native builds; using a combination of the `bellsoft-liberica`, `spring-boot` and `native-image` buildpacks.
```

Comment:
```suggestion
Since first-class native builds were added in Spring Boot 3.0, Paketo buildpacks support Spring Boot native builds; using a combination of the `bellsoft-liberica`, `spring-boot` and `native-image` buildpacks.
```

Mostly the `Since Spring Boot 3.0,...` part gave me pause. I know that's when we added official support, but we were doing builds before that. Not sure if that level of detail matters, but may throw off some readers. Suggestion is to focus on the official support being added in 3.0.